### PR TITLE
#162471395 Implement tests for Progress components

### PR DESCRIPTION
**Pull request by emmabaye**
#### What does this PR do?
Adds test for Progress components and raise the coverage to a minimum of 80%
#### Description of Task to be completed?
N/A
#### Where should the reviewer start?
[Here](https://github.com/AndelaOSP/wire/commit/173ad4d34e1061d1dd7aabc293d15d9d037b2203)
#### How should this be manually tested?
Run command `npm run test`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#162471395](https://www.pivotaltracker.com/story/show/162471395)
#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: src/Components/Progress/Progress.test.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
```
[the remark this answers is not in the document]
Comment (theartisancodes) [reply src/Components/Progress/Progress.test.jsx]: fix all lint issues

[the remark this answers is not in the document]
Comment (emmabaye) [reply src/Components/Progress/Progress.test.jsx]: I have fixed the lint issues. The lint issues can't be found in the code

**Review by theartisancodes — COMMENTED** [review]
I think you can still add more tests on this. You can check if it renders the material-ui component. and also the className

**Comment by theartisancodes** [comment]
why the decrease in test coverage?

**Comment by emmabaye** [comment]
I will look into it

**Comment by emmabaye** [comment]
> why the decrease in test coverage?

I have fixed it

**Pull request merged**